cdp_scrapers/legistar_utils.py

```diff
@@ -473,6 +488,27 @@ def get_video_uris(self, legistar_ev: Dict) -> List[Dict]:
         )
         raise NotImplementedError
 
+    def get_time_zone(self) -> str:
+        """
+        Return time zone name for CDP instance.
+        To use dynamically determined time zone,
+        use find_time_zone().
+
+        Returns
+        -------
+        time zone name : str
+            i.e. "US/Pacific" | "America/Los_Angeles" | ...
+
+        See Also
+        --------
+        SeattleScraper.get_time_zone()
+
+        """
+        log.error(
+            "time zone name e.g. US/Pacific " "required for proper event timestamping"
+        )
+        raise NotImplementedError
```

Comment:
I am tempted to say that we should just use the function you wrote down below... the "find_time_zone" function as a default?

Reply:
I considered that initially, but didn't know if all instances are run within same OS or in virtual guests or something similar. If the latter, then I agree; since the OS can be configured to have local clock appropriate for that municipality. If the former, wouldn't every instances end up using PT?...

Reply:
Ohhhhhh right right right. Okay good point. I guess we should add a field to the cookiecutter that asks for timezone.

Reply:
Unless someone is doing a backfill locally, all the instances run on UTC clocked GitHub Action workers.